app/championship_setup/championship_class.py

```diff
@@ -0,0 +1,77 @@
+from app.knights_setup.knight_class import Knight

```

Comment:
Rename module Instead of `championship_setup` to `championship`, it could have files that are used for other purposes than the setup in the future.